catalog/imaging/imaging-transformer-nitf/src/main/java/org/codice/alliance/transformer/nitf/common/HistoaAttribute.java

```diff
@@ -0,0 +1,95 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.alliance.transformer.nitf.common;
+
+import java.io.Serializable;
+import java.util.Collections;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.function.Function;
+
+import org.codice.imaging.nitf.core.tre.Tre;
+
+import ddf.catalog.data.AttributeDescriptor;
+import ddf.catalog.data.AttributeType;
+import ddf.catalog.data.impl.BasicTypes;
+
+/**
+ * TRE for "Softcopy History Tagged Record Extension"
+ */
+public class HistoaAttribute extends NitfAttributeImpl<Tre> {
+
+    private static final List<NitfAttribute<Tre>> ATTRIBUTES = new LinkedList<>();
+
+    static final String SYSTYPE_NAME = "SYSTYPE";
+
+    static final String PC_NAME = "PC";
+
+    static final String PE_NAME = "PE";
+
+    static final String REMAP_FLAG_NAME = "REMAP_FLAG";
+
+    static final String LUTID_NAME = "LUTID";
+
```

Comment:
What about NEVENTS? `This field shall contain the number of processing events associated with the image.`